README.md

```diff
@@ -16,3 +16,13 @@ This repository is licensed under CC0 1.0 Universell (CC0 1.0) Public Domain Ded
 Please read the [wiki](https://github.com/OpenEnergyPlatform/ontology/wiki) for information about the ontology and its standards.
  
 The workflow is described in the [CONTRIBUTE.md](https://github.com/OpenEnergyPlatform/ontology/blob/dev/CONTRIBUTE.md) file. Please check it out **before** you start working on this project. Points that are not clear in the file can be discussed in a [github issue](https://github.com/OpenEnergyPlatform/ontology/issues/new/choose).
+
+## Teams tag
+
+If you have a specific question about ontology, modelling or economy (in context of the ontology), you can [tag](https://github.com/OmahaGirlsWhoCode/OmahaGirlsWhoCode/wiki/How-to-tag-someone-in-a-pull-request) one of these team to notify its members that you need their feedback/help.
+
+- @economy-experts
+
+- @ontology-experts
+
+- @modelling-experts
```

Comment:
The three groups are mentioned here. Could you please also add (clickable) links?

Reply:
Done in bf47c6882a650515d858d131f7ecc1e39a60dbdb :)